TTS/enhancer/layers/losses.py

```diff
@@ -0,0 +1,91 @@
+from typing import Type
+import torch
+from TTS.utils.audio import TorchSTFT
+from TTS.tts.layers.losses import L1LossMasked
+from TTS.vocoder.layers.losses import MelganFeatureLoss, MSEGLoss, MSEDLoss, _apply_G_adv_loss, _apply_D_loss
+
+class BWEGeneratorLoss(torch.nn.Module):
+    def __init__(self, device):
+        super().__init__()
+        self.l1_masked= L1LossMasked(False)
+        self.specs = [TorchSTFT(
```

Comment:
Maybe you can replace it with the loss_mag of the MultiScaleSTFTLoss, in this way you can reuse the code  and have the loss normalized.

Reply:
Good idea, I'll try that